Add: 1.11.0-beta2 announcement (#193)
OpenTTD/website: _posts/2021-02-28-openttd-1-11-0-beta2.md

---
title: OpenTTD 1.11.0-beta2
author: TrueBrain
---

Today we release our second beta of our 1.11 release series!

> This second beta is a huge one: we spend the time between last beta and this on massively improving the performance of our video-drivers on all platforms.

Reviewer: ```suggestion
This second beta is a huge one: we have spent much of the time between last beta and this on massively improving the performance of our video-drivers on all platforms.
```

With a huge thanks to michi_cc, we now even have OpenGL support!
Don't get your hopes up: it still is (and always will be) a 2D game.

To make sure these changes get their fair share of testing, we ask you to help us test them. Especially the following new features:
* OpenGL support for all platforms.
* Massive improvements to all video-drivers; so massive, we now have a setting to indicate an upper-limit to fast-forward speeds.

> * Display now runs on 60fps.

Reviewer: "or higher, if you alter the refresh-rate setting in the config file" ?

Author: Let's for now keep this a secret :)

> * Generic linux builds.

Reviewer: ```suggestion
* Generic Linux builds.
```

For a complete list of changes, please see the changelog.

Additionally, we are still searching for translators for 19 languages! 

> Below is the list of language that could really use a translator or two.

Reviewer: ```suggestion
Below is the list of languages that could really use a translator or two.
```

> If you can read/write any of these language, and are willing to help us out make the game look better in that language, please drop by https://translator.openttd.org/ and help us out!

Reviewer: ```suggestion
If you can read/write any of these languages, and are willing to help us out make the game look better in that language, please drop by https://translator.openttd.org/ and help us out!
```

We made things as easy as possible for you; all we need from you is a few minutes of your time to translate strings from English to your language.

With this new beta, we are slowly wrapping up 1.11, meaning we are closing in on a feature-freeze: a period of time we do not accept new features, but focus on any and all bugs we are aware of.
This means it is a perfect time for you to report any you have!

Please do help us test, and let us know how you like the improvements we made!

* [Download](https://www.openttd.org/downloads/openttd-releases/testing.html)
* [Changelog](https://cdn.openttd.org/openttd-releases/1.11.0-beta1/changelog.txt)
* [Bug tracker](https://github.com/OpenTTD/OpenTTD/issues)

> Language without translator:

Reviewer: ```suggestion
Languages without translators:
```
